package.json

```diff
@@ -52,6 +52,10 @@
 			{
 				"title": "Biome: Troubleshoot - Reset",
 				"command": "biome.reset"
+			},
+			{
+				"title": "Biome: Initialize Workspace",
```

Comment:
Perhaps just "Initialize". This would feel more aligned with the other commands (which are quite terse) and avoid confusion with "workspace" in the package management sense (users may be hesitant to run this in a non-monorepo context).

Reply:
renamed